libcudacxx/include/cuda/std/__cccl/dialect.h

```diff
@@ -104,6 +105,11 @@
 #  define _CCCL_NO_VARIABLE_TEMPLATES
 #endif // _CCCL_STD_VER <= 2011
 
+// concepts are only available from C++20 onwards
+#if _CCCL_STD_VER <= 2017 || !defined(__cpp_concepts) || (__cpp_concepts < 201907L)
+#  define _CCCL_NO_CONCEPTS
```

Comment:
Question: Should we move towards values so that we can skip the `defined` Those are internal macros so it will give the right answer nevertheless
```suggestion
#  define _CCCL_NO_CONCEPTS 1
```

Reply:
Yes and I think these should also be function-like macros for the same reasons as the compiler detection macros

Reply:
... but i don't think this is the right PR to do that.